globals.go

```diff
@@ -263,6 +264,8 @@ func (k SenderKey) String() string {
 		return "jira-issue"
 	case SenderGithubMerge:
 		return "github-merge"
+	case SenderCommitQueueDequeue:
```

Comment:
Is this the same thing as a merge?

Reply:
I decided to make a separate sender for the server commit queue whose whole job is to dequeue the item from the queue.

Reply:
Got it, thanks.